.buildkite/pipeline.yml

```diff
@@ -498,16 +498,16 @@ steps:
 ##### Regression
 #####
 
-  - label: "🦝 gpu shallow water regression tests"
-    env:
-      JULIA_DEPOT_PATH: "$SVERDRUP_HOME/.julia-$BUILDKITE_BUILD_NUMBER"
-      TEST_GROUP: "shallowwater_regression"
-    commands:
-      - "$SVERDRUP_HOME/julia-$JULIA_VERSION/bin/julia -O0 --color=yes --project -e 'using Pkg; Pkg.test()'"
-    agents:
-      queue: Oceananigans
-      architecture: GPU
-    depends_on: "init_gpu"
+  # - label: "🦝 gpu shallow water regression tests"
+  #   env:
+  #     JULIA_DEPOT_PATH: "$SVERDRUP_HOME/.julia-$BUILDKITE_BUILD_NUMBER"
+  #     TEST_GROUP: "shallowwater_regression"
+  #   commands:
+  #     - "$SVERDRUP_HOME/julia-$JULIA_VERSION/bin/julia -O0 --color=yes --project -e 'using Pkg; Pkg.test()'"
+  #   agents:
+  #     queue: Oceananigans
+  #     architecture: GPU
+  #   depends_on: "init_gpu"
```

Comment:
Does this PR touch any code that affects this regression test? @simone-silvestri can you comment?